coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
+    aspects = defaultdict(lambda: [])
```

Comment:
@pratyushprakash @adhikasp Thanks :) Good points! I chose the way of consistently using `defaultdict`: https://github.com/coala/coala/pull/3966/files#diff-7544b01c348d9f3cfa526b33644dc42aR25